### PR TITLE
Modify add member and add owner modals to provide a warning when a department account is being added.

### PR DESCRIPTION
**Pull request by mikhail-shkaralevich**
# Ticket Link

[Ticket](https://uhawaii.atlassian.net/browse/GROUPINGS-1706)

# List of squashed commits

- Modify add member and add owner modals to provide a warning when a department account is being added.
- Add jasmine tests.

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [x] Project Jasmine Tests Passed:
- [x] Executes Expected Functionality:
- [x] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

File: src/main/resources/static/javascript/mainApp/grouping.controller.js [hunk 1]
```diff
@@ -809,12 +810,22 @@
...
             $scope.listName = options.listName;
             $scope.isMultiAdd = uhIdentifiers.length > 1;
 
+            // Check if there is a dept account to be added.
+            for (const member of membersAttributesResults) {
+                if (member['uid'] === member['uhUuid']) {
+                    $scope.hasDeptAccount = true;
+                    break;
+                }
+                $scope.hasDeptAccount = false;
```
Comment (fduckart) [inline src/main/resources/static/javascript/mainApp/grouping.controller.js:824]: Maybe move this above the for-statement?

**Comment by JorWo** [comment]
From a design perspective I would remove the "Warning:" from the front of the message so it matches the warning message below it: 
image
Both warning messages do not start with "Warning:"

**Review by mhodgesatuh — APPROVED** [review]

File: src/main/resources/static/javascript/mainApp/grouping.controller.js (hunk 1 above)
Comment (mikhail-shkaralevich) [reply src/main/resources/static/javascript/mainApp/grouping.controller.js:824]: Right, that's a better idea.

**Review by JorWo — APPROVED** [review]
Great work

**Pull request merged**